e2e/tests/functional/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -211,7 +215,11 @@ test.describe('Example Imagery Object', () => {
     // steps not working for me here
     await page.mouse.move(canvasCenterX - 20, canvasCenterY - 20);
     await page.mouse.move(canvasCenterX - 100, canvasCenterY - 100);
+    // toolbar should hide when we're creating annotations with a drag
+    await expect(page.locator('[role="toolbar"][aria-label="Image controls"]')).toBeHidden();
```

Comment:
This is perfect. I wonder if there's a way to update the testcase title to reflect this change? Nothing comes to mind